dev/com.ibm.websphere.org.eclipse.microprofile/build.gradle

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+configurations {
+  config11 {
+    canBeResolved = true
+  }
+  config121 {
+    canBeResolved = true
+  }
+  config13 {
+    canBeResolved = true
+  }
+  config14 {
+    canBeResolved = true
+  }
+  ft10 {
+    canBeResolved = true
+  }
+  ft11 {
+    canBeResolved = true
+  }
+}
+
+artifacts {
+  config11(file('build/libs/com.ibm.websphere.org.eclipse.microprofile.config.1.1.jar'))
+  config121(file('build/libs/com.ibm.websphere.org.eclipse.microprofile.config.1.2.1.jar'))
+  config13(file('build/libs/com.ibm.websphere.org.eclipse.microprofile.config.1.3.jar'))
+  config14(file('build/libs/com.ibm.websphere.org.eclipse.microprofile.config.1.4.jar'))
+  ft10(file('build/libs/com.ibm.websphere.org.eclipse.microprofile.faulttolerance.1.0.jar'))
+  ft11(file('build/libs/com.ibm.websphere.org.eclipse.microprofile.faulttolerance.1.1.jar'))
+}
```

Comment:
I can see that this is solving the problem of exposing the sub-bundles to fat tests which want to depend on them.

However, it looks a little at odds with what's in the Gradle user guide here: https://docs.gradle.org/current/userguide/declaring_dependencies.html#sec:resolvable-consumable-configs

The user guide suggests:
 * `canBeResolved` and `canBeConsumed` both default to `true`.
 * for a configuration exposed to other projects as a dependency, you want `canBeResolved = false` and `canBeConsumed = true`

I'm not necessarily saying this is wrong, since I'm not familiar enough with gradle (and I wouldn't be surprised if we were doing something a bit weird), but thought it worth mentioning that it looks different to what the user guide suggests.

Reply:
Yes I struggled some with what to set them to.  I will say it is working the way that I have it configured.  Tests are getting just the bundle they are after.  I have asked @fritze2 to take another look at the PR in light of these new gradle file changes to get his feedback since he has a lot more experience with using gradle.

Reply:
Yes, I'd expect it to work since `canBeConsumed` will default to `true`.

Reply:
I read through what you pointed to in the guide and updated it to consume = true and resolve = false based off of my reading.  I ran one of the FATs with the change and it worked just fine.  With consume = false it failed miserably, but resolve = false looks to be what we want from reading through things.  The PR commit 1 is updated with the change.

Reply:
Looks good, thanks.